charts/opentelemetry-target-allocator/values.yaml

```diff
@@ -14,21 +14,108 @@ nameOverride: ""
 fullnameOverride: ""
 
 targetAllocator:
+  # Number of replicas for the target allocator deployment
+  replicaCount: 1
+
   image:
     repository: ghcr.io/open-telemetry/opentelemetry-operator/target-allocator
     # The tag of the Target Allocator image, default value is the chart appVersion
     tag: ""
+    # Image pull policy
+    pullPolicy: IfNotPresent
+
   # Secrets to attach to the respective serviceaccount to pull docker images
   imagePullSecrets: []
+
   serviceAccount:
     # Specifies whether a ServiceAccount should be created
     create: true
     # The name of the ServiceAccount to use.
     # If not set and create is true, a name is generated using the fullname template
     name: ""
-
     # Service account annotations
     annotations: {}
+
+  # Security context for the pod
+  podSecurityContext:
+    runAsNonRoot: true
+    runAsUser: 65534
+    runAsGroup: 65534
+    fsGroup: 65534
+    seccompProfile:
+      type: RuntimeDefault
```

Comment:
@atoulme @jaronoff97 can you confirm these default security settings work for the target allocator?